### DISCUSSION TITLE
What are some of the ideas behind the language keywords reserved for future use?

### DISCUSSION THREAD

**Discussion by ev-dev** [PowerShell/PowerShell#18069] [General]
Hey pwsh fans and devs!

After using PowerShell for years, I've only now just come across the docs for `about_Reserved_Words` and noticed that 3 of the keywords were marked as reserved for future use:
-  `define`
- `from`
- `var`

All 3 of these seem intriguing in their own right, but as you might imagine, they are very difficult to try and search for on the interwebs due to their generality as English words. It's also possible that there really isn't any information/discussion about these reserved-for-future-use keywords out there publicly. 

Anyone know some of the motivations/ideas around these future words?

> Link to docs page:
> https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_reserved_words

> _PS. Not sure if this is the best place to ask this kind of conceptual/roadmap question but figured I'd try_

**Comment by travisclagrone** [comment]
Although I am not privy to the original reasoning, I would imagine they will be kept reserved even if never implemented simply in order to prevent confusion / clashing expectations when coming from other languages.
